Library/Contributions/pre-push

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+# Makes sure bottles are fetchable after BrewTestBot modifies a formula.
+# Only checks pushes to master on Homebrew/homebrew* remotes.
+# To use, `ln -s $(brew --repository)/Library/Contributions/pre-push
+# $(brew --repository)/.git/hooks`.
+
+import re
+import os
+import subprocess
+import sys
+
+FORMULA_RE = re.compile(r"^(Library/Formula/)?([A-Za-z0-9-]+)\.rb")
+TESTBOT_EMAILS = {"brew-test-bot@googlegroups.com"}
+
+def git(*args):
+    return subprocess.check_output(["git"] + list(args)).strip()
```

Comment:
I felt the same way but `*args` gives you a tuple and you can't add a list and a tuple. You _can_ add two tuples, though, so this should be `("git",)` instead of `["git"]`.